src/core/geometry/bundle/Bundle.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <string>
+#include "core/util/Identification.h"
+#include "core/geometry/element/Element.h"
+
+namespace semba::geometry::bundle {
+	typedef geometry::ElemId ElemIdForBundle;
```

Comment:
I am preferring to use 
`using ElemIdForBundle = geometry::ElemId;`
it has some benefits when used with templates, but nothing too serious tbh.

Reply:
Addressed in commit e0876b681447e7be6b8220111352cf1fac6c7c78.